modules/mobile/docs/openapi.yaml

```diff
@@ -396,6 +396,89 @@ paths:
       security:
         - Bearer: []
       summary: /v0/appointments/cancel/{cancelId}
+  /v0/appointments/requests:
```

Comment:
Looks good! Docs just need to be updated to match the new route `/v0/appointment`

Reply:
Good catch